docs/topics/mta-cli-analyze.adoc

```diff
@@ -25,27 +25,42 @@ Usage:
   {mta-cli} analyze [flags]
 
 Flags:
-      --analyze-known-libraries   analyze known open-source libraries
-  -h, --help                      help for analyze
-  -i, --input string              path to application source code or a binary
-      --json-output               create analysis and dependency output as json
-      --list-sources              list rules for available migration sources
-      --list-targets              list rules for available migration targets
-  -l, --label-selector string     run rules based on specified label selector expression
-      --maven-settings string     path to a custom maven settings file to use
-      --overwrite                 overwrite output directory
-      --skip-static-report        do not generate the static report
-  -m, --mode string               analysis mode. Must be one of 'full' or 'source-only' (default "full")
-  -o, --output string             path to the directory for analysis output
-      --rules stringArray         filename or directory containing rule files
-      --skip-static-report        do not generate the static report
-  -s, --source string             source technology to consider for analysis. To specify multiple sources, repeat the parameter: --source <source_1> --source <source_2> etc.
-  -t, --target string             target technology to consider for analysis. To specify multiple targets, repeat the parameter: --target <target_1> --target <target_2> etc.
+      --analyze-known-libraries             Analyze known open-source libraries.
+      --context-lines int                   Number of lines of source code to include in the output for each incident (default: 100).
+  -d, --dependency-folders stringArray      Directory for dependencies.
+      --enable-default-rulesets             Run default rulesets with analysis (default: true).
+  -h, --help                                Help for analyze.
+      --http-proxy string                   HTTP proxy string URL.
+      --https-proxy string                  HTTPS proxy string URL.
+      --incident-selector string            An xxpression to select incidents based on custom variables. Example: (!package=io.konveyor.demo.config-utils).
```

Comment:
Better an `xxpression` than an `XXXpression`, I guess... you can make your own joke

Reply:
Joke's on me. I didn't push my last commit.